apps/api/src/applicant/dto/ienapplicant-filter.dto.ts

```diff
@@ -10,12 +10,12 @@ export class IENApplicantFilterAPIDTO extends IENApplicantFilterDTO {
   @IsString()
   ha_pcn?: string;
 
-  @ApiPropertyOptional({
-    description: 'Provide optional status(with comma separated values e.g. 9,10,11)',
-  })
-  @IsOptional()
-  @IsString()
-  status?: string;
+  // @ApiPropertyOptional({
+  //   description: 'Provide optional status(with comma separated values e.g. 9,10,11)',
+  // })
+  // @IsOptional()
+  // @IsString()
+  // status?: string;
```

Comment:
This field would be used in the main page. I am just curious why you commented out this.

Reply:
Some queries are there that need to be solved about status/milestone import data for applicants. So I might need to change its schema and relation, so it won't work with the existing setup. So I have disabled the status/milestone filter, for now,

Reply:
I'll try to re-enable it early next week. Mostly next Monday itself.